### ISSUE TITLE
Suppress unnecessary Chrome translation prompt

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#12647]
![image](https://user-images.githubusercontent.com/4071474/121571433-de520180-c9d7-11eb-9b29-825fe4ff324c.png)

(image was shared with me, I tried the particular URL but did not repro)

Perhaps adding a `lang` attribute to the body will clear this up, dynamically set to the LHR's language. Same for report renderer (but set to the report root container).

**Comment by adamraine** [comment]
I remember reproing this a couple times. Pretty sure `lang` on `<html>` would resolve this.

**Comment by alankent** [comment]
Just FYI, I created the above screenshot. I opened the URL in a new tab and did not get the same issue. I got to the page from the PageSpeed Insights page button (so it had a referrer header I guess). Interestingly, I got another icon in the URL in addition to the "Google Translate" button.

![image](https://user-images.githubusercontent.com/5702163/121575678-7d78f800-c9dc-11eb-873e-991e8b676a3f.png)

Whereas the page without the translation issue showed

![image](https://user-images.githubusercontent.com/5702163/121575723-8d90d780-c9dc-11eb-8932-85a57980a21a.png)

Reloading the page with the Maltese message still shows the message after reload. The etag for the two tabs was the same, the same cache server was listed. The only HTTP header I could see being significant and different was the referrer header. E.g. both had "accept-language: en-US,en;q=0.9" as HTTP headers.

Basically it appears Chrome guessed the language of the page and got it wrong on one guess (then stuck with that guess in the tab). Adding language into the HTML markup makes sense to improve the guess accuracy. I cannot see any errors in the existing HTML markup or HTTP headers. There is no "Maltese" language code or anything like that I can see in the page or HTTP headers (mt_MT).
